app/AppKernel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Component\Config\Loader\LoaderInterface;
+
+class AppKernel extends Kernel
```

Comment:
__[code_sniffer]__
* Missing class doc comment
* Each class must be in a namespace of at least one level (a top-level vendor name)

<!-- Comment by [SideCI](https://sideci.com) -->